create-ladder.js

```diff
@@ -0,0 +1,20 @@
+const showLadder = function(){
+  $('#ladder').addClass('opened');
+  var folds = $('.fold', '#ladder'); console.log(folds);
+  unfoldQ(folds,0);
+
+  function unfoldQ(elem,inx){
```

Comment:
Why is this function defined inside show ladder?

Also, in one place you have used `const x = function()`, in another `function x()`

Its inconsistent.